specification/cognitiveservices/data-plane/FormRecognizer/preview/v1.0/FormRecognizer.json

```diff
@@ -296,6 +296,26 @@
           "maxLength": 2048,
           "minLength": 0,
           "type": "string"
+        },
+        "sourceFilter":{
```

Comment:
Should update samples to show the usage.

Reply:
Done.